### ISSUE TITLE
TransformerBlock undefined

### ISSUE THREAD

**Issue by dawnlh** [xuboming8/DSTNet#7]
https://github.com/xuboming8/DSTNet/blob/726293de5e1a2a5ddc0ddb8cc996aef12c3a8640/basicsr/archs/ChanDynamic_GMLP.py#L94C27-L94C27

"TransformerBlock" in this line is undefined, it seems to be "CWGDN".

**Comment by xuboming8** [comment]
Thanks , I have fixed this bug.

**Comment by dawnlh** [comment]
By the way, does the pre-trained model can only deal with images with the same size as the training data? I try to use the GoPro pretrained model for testing on the resized (x0.5, 640*360) images due to limited GPU memory, but get the following error:

```
  File "/hdd/0/zzh/project/INRT/DSTNet/basicsr/archs/kpn_pixel.py", line 235, in _idynamic_cuda
    assert input.size(-2) // stride == weight.size(-2)
```

So, I must train a new model for images with different sizes?

Thanks.

**Comment by xuboming8** [comment]
The patch size used for training and test can be different. I used 256 * 256 patch for training and full size for test.
When you use different patch size, you need to pay attention to whether it is a multiple of 16, obviously 360 is not.You can add padding to the input video to a multiple of 16 during test.
Due to limited GPU memory,you can reduce the number of frames during inference.

**Comment by dawnlh** [comment]
Get it! Thanks for your patient explanation.
